WebRoot/css/zm.css

```diff
@@ -3906,15 +3906,43 @@ HTML>BODY .appt-selected .appt_allday_body
 	height:auto;
 }
 
-.ZmPrefZimletListView .DwtListView-Rows {
-	overflow:visible;
+.ZmPrefZimletDetail {
+	@PrefZimletDetail@
 }
 
-.ZmPrefZimletListView .Row {
-	overflow:hidden;
+.ZOptionsItemsListView {
+	@PanelBorder@
+}
+
+.ZOptionsItemsListView .DwtListView-ColHeader {
+	@OptionsItemsListColumnHeaderWrapper@
 }
 
-.ZmPrefZimletListView .Row-selected {
+.ZOptionsItemsListView .DwtListView-Column {
+	@OptionsItemsListColumnHeader@
```

Comment:
This is for list header item cell. Changed the name to OptionsItemsListColumnHeaderCell, 
https://github.com/Zimbra/zm-web-client/pull/98/commits/6a6a14dd2f05f961c3d5d8d962c046d40c19d080